src/Shared/Recruit.Shared.Web/ViewModels/Skills/SkillsViewModelHelper.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.Client;
+
+namespace Esfa.Recruit.Shared.Web.ViewModels.Skills
+{
+    public class SkillsViewModelHelper
+    {
+        private const int ColumnOneCutOffIndex = 9;
+        private const char SortPrefixSeparator = '-';
+
+        private readonly Lazy<List<string>> _lazyCandidateSkills;
+        
+        private List<string> CandidateSkills => _lazyCandidateSkills.Value;
+        private IEnumerable<string> Column1BuiltInSkills => CandidateSkills.Take(ColumnOneCutOffIndex);
+        private IEnumerable<string> Column2BuiltInSkills => CandidateSkills.Skip(ColumnOneCutOffIndex);
+
+        public SkillsViewModelHelper(IRecruitVacancyClient vacancyClient)
```

Comment:
Could we not use a passed in func delegate to avoid having this class knowing about the VacancyClient.

Reply:
👍